main.tex

```diff
@@ -1429,50 +1440,42 @@
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must not affix a sign to its Lot or to the Common Property unless the sign:
+\item  An Owner or Occupier of a Residential Lot must not affix a sign to their Lot (including their windows) unless the sign:
 
 \begin{enumerate}[label=(\alph*)]
 
 \item  complies with all legislative requirements including any planning permits and requirements of local council and relevant authorities;
 
-\item  has been approved by the Owners Corporation; and
-
-\item  signage proposals must be submitted to the Owners Corporation for approval.
+\item is not for commercial purposes (including real estate)
 
 \end{enumerate}
 
+\item An Owner or Occupier of a Lot must not affix a sign to Common Property unless the sign has been approved by the Owners Corporation
+
+
 \item  An Owner or Occupier of a Lot must:
 
 \begin{enumerate}[label=(\alph*)]
 
 \item  make good any damage caused by the installation or removal of any sign affixed to its Lot or to the Common Property and restore the Common Property to the condition it was in before the sign was installed;
 
-\item  not display handwritten signs or notices which are visible from the exterior of their Lot;
-
-\item  not display a self-standing sign such as A Frames outside the Lot boundary;
```

Comment:
If you live in Brunswick Yard and you run a café in Coburg, you are not allowed to put an A-Frame outside your café.